ingress/controllers/nginx/README.md

```diff
@@ -0,0 +1,184 @@
+# Nginx Ingress Controller
+
+This is a nginx Ingress controller that uses [ConfigMap](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/configmap.md) to store the nginx configuration. See [Ingress controller documentation](../README.md) for details on how it works.
+
+
+## What it provides?
+
+- Ingress controller
+- nginx 1.9.x with
+- SSL support
+- custom ssl_dhparam (optional). Just mount a secret with a file named `dhparam.pem`.
+- support for TCP services (flag `--tcp-services-configmap`)
+- custom nginx configuration using [ConfigMap](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/configmap.md)
+- custom error pages. Using the flag `--custom-error-service` is possible to use a custom compatible [404-server](https://github.com/kubernetes/contrib/tree/master/404-server) image
+
+
+## Requirements
+- default backend [404-server](https://github.com/kubernetes/contrib/tree/master/404-server)
+
+
+
+## Create a loadbalancer
+
+Loadbalancers are created via a ReplicationController or Daemonset
+
+First we need to deploy some application to publish. To keep this simple we will use the [echoheaders app](https://github.com/kubernetes/contrib/blob/master/ingress/echoheaders/echo-app.yaml) that just returns information about the http request as output
+```
+kubectl run echoheaders --image=gcr.io/google_containers/echoserver:1.1 --replicas=1 --port=8080
```

Comment:
We have echoserver 1.2 and 1.3, use one of those (see README of echoserver for differences)?